src/ctia/stores/es/crud.clj

```diff
@@ -40,6 +41,44 @@
   (let [[orig docid] (re-matches #".*?([^/]+)\z" id) ]
     docid))
 
+(defn remove-es-actions
+  "Removes the ES action level
+
+  [{:index {:_id \"1\"}}
+   {:index {:_id \"2\"}}]
+
+  ->
+
+  [{:_id \"1\"}
+   {:_id \"2\"}]
+  "
+  [items]
+  (map (comp val first) items))
```

Comment:
I found it hard to read because I'm really not used to `val` and also I generally never user `first` directly on a map to transform a single element hash-map as a list of two elements [k v].

Would you mind to use `(map (comp first vals) items)` instead?